templates/tab/js/default/public/auth-start.html

```diff
@@ -23,30 +24,29 @@
         localStorage.removeItem("codeVerifier");
```

Comment:
since we use MSAL now, can you see whether some legacy code (e.g. the state and codeVerifier part) can be cleaned?

Reply:
All useless code removed. THx!